docs/SERVER_CONFIGURATIONS.md

```diff
@@ -415,7 +415,7 @@ The replication factor for change log topics and repartition topics created by t
 The number of partitions in each internal kafka topic. Disabled if `LHS_SHOULD_CREATE_TOPICS=false`.
 
 - **Type:** int
-- **Default:** 72
+- **Default:** 12
 - **Importance:** low
```

Comment:
This config is High importance actually. The server actually needs to know how many partitions there are in order to properly guess partitions of stuff for the metadata partition, even if create topics is false.

I can fix that in my PR after we merge this one.